content/efcore/getting-started/what-is-ef-core.md

```diff
@@ -0,0 +1,5 @@
+# What is Entity Framework Core?
+
+Entity Framework Core (EF Core) is a lightweight, extensible, and cross-platform version of Entity Framework. Entity Framework is an object-relational mapper (O/RM) that enables .NET developers to work with a database using .NET objects. This allows developers to create and manage databases without most of the data-access code that developers usually need to write. 
```

Comment:
How do you feel about this flow:
1. Describe EF
2. Describe EF Core
3. Compare EF Core to EF 6.x